iFixit/vigilo
# Feature: Add a Helm Chart for Vigilo

## Description

This mainly follows examples I've seen from other Helm Charts. The entire premise to be able to routinely kick off a lighthouse audit as part of our Kubernetes cluster. This will detach the dependency of having Lighthouse running on our Ubreakit machine and instead will be ran through a Pod. 

As such, it will be a lot easier if we have a Helm Chart for all this instead of manually creating the manifests ourselves in our FluxCD repo.

### QA 

> [!Note]
> This will be difficult to QA as is, we will need to manually test by integrating the Helm Chart into our current Test cluster.

1. Declare a new `GitRepository` FluxCD resource
2. Define a Helm Release
	- Use the AWS ECR repo in the values section
4. Check if the CronJob worked successfully.


Closes: https://github.com/iFixit/vigilo/issues/16
Connects: https://github.com/iFixit/ifixit/issues/49455
